packages/ui/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "private": true,
```

Comment:
This should be removed to publish `@aws-amplify/ui`.

Reply:
```suggestion
```